ta/pkcs11/src/object.h

```diff
@@ -56,4 +56,21 @@ enum pkcs11_rc entry_create_object(struct pkcs11_client *client,
 
 enum pkcs11_rc entry_destroy_object(struct pkcs11_client *client,
 				    uint32_t ptypes, TEE_Param *params);
+
+enum pkcs11_rc entry_find_objects_init(struct pkcs11_client *client,
+				       uint32_t ptypes, TEE_Param *params);
+
+enum pkcs11_rc entry_find_objects(struct pkcs11_client *client,
+				  uint32_t ptypes, TEE_Param *params);
+
+enum pkcs11_rc entry_find_objects_final(struct pkcs11_client *client,
+					uint32_t ptypes, TEE_Param *params);
+
+enum pkcs11_rc entry_get_attribute_value(struct pkcs11_client *client,
+					 uint32_t ptypes, TEE_Param *params);
+
+enum pkcs11_rc entry_get_object_size(struct pkcs11_client *client,
+				     uint32_t ptypes, TEE_Param *params);
+
+void release_session_find_obj_context(struct pkcs11_session *session);
```

Comment:
add empty line between `release_session_find_obj_context()` and `#endif`.

Reply:
> @ruchi393 please note that I have updated wip branch with new commits and updated the comment above.

Thanks @vesajaaskelainen . I was thinking I will work on the comments in this PR first. Once this PR is merged,  you can raise another PR with fixes in the serialization logic on your wip branch. Does that sound reasonable ?

Reply:
I am OK with that.